components/Tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import { cn } from "helpers";
+
+export enum TabsSize {
+  Default = "default",
+  Large = "large",
+}
+
+export interface TabsProps {
+  items: Array<{
+    label: string | React.ReactNode;
+    onClick: () => void;
+    selected?: boolean;
+  }>;
+  size?: TabsSize;
+}
+
+export default function Tabs_old({
```

Comment:
Currently the homepage still uses these Tabs via `FiltersPicker`, which went unchanged because it imports via an import alias ([this](https://stackoverflow.com/questions/39282253/how-can-i-alias-a-default-import-in-javascript)). I want to replace these but, but I'll get to it in this PR; I want to make sure the new Tabs (in `components/ui/tabs`) can be used in both places, probably by styling a new variant.

On leaving it, I thought this weird naming would help us know it's deprecated. But k, I also just used the `@deprecated` JSDoc tag. This makes it so that you'll get a hardy warning in intellisense. Curious if you're cool with this? Or should I just leave them as `Tabs` and the deprecated tag is enough?

<img width="970" alt="image" src="https://github.com/hawaiians/hawaiiansintech/assets/5141111/0ecc2956-55f3-422f-b5a6-b189a69e7646">

Reply:
gotcha no worries yea if it's still being used then thats all good. makes sense to defer the replacement to another PR. I don't really mind either way in terms of the `@deprecated` tag or not